### DISCUSSION TITLE
[PickerInput]: how to provide your own render for row

### DISCUSSION THREAD

**Discussion by mamuna36** [epam/UUI#453] [Q&A] [answered]
**Description**
I am utilizing the PickerInput component and what we want to see on our page is that when the user starts typing he should see a filtered list of options where the text typed by the user is emphasized. With the current setup, I can not use a React.Element as the name property in the array of options.

**Comment by AlekseyManetov** [comment]
Hi, could you please add the screenshots or designs of your expected PickerInput behaviour?

**Comment by mamuna36** [comment]
I tried to attach a file but could not do so. Here is a link to the design https://www.figma.com/file/PvZO7XCdPEsvQaLFfVYcut/Untitled?node-id=0%3A1

please let me know if you can open it. Moreover, we need it with multiSelection mode.

**Comment by AlekseyManetov** [comment, marked as answer]
You need to use `renderRow` prop to provide your own row render, see the example here - 
https://uui.epam.com/documents?category=components&id=pickerInput&mode=doc&skin=UUI4_promo#custom_picker_row

**Comment by mamuna36** [comment]
thanks for this mention.
